software/context.qmd

```diff
@@ -1,15 +1,51 @@
 ---
-title: System context and scope
+title: System Context and Scope
 order: 3
 ---
 
 {{< include /includes/_wip.qmd >}}
 
-This document is an overview of the different connections, both human
-and computer, with a [Data
-Resource](introduction.qmd#definition-of-terms) managed by the Seedcase
-framework.
+This section establishes the system boundaries of the Seedcase framework
+and situates the framework in its operational context by describing its
+potential human and computer communication partners. Human communication
+partners are grouped into user roles which are broadly applicable to the
+entire Seedcase framework. For a detailed discussion of these user
+roles, see [User Requirements](user-requirements.qmd). Whether any
+computer communication partners are present for a particular Seedcase
+installation depends on the specific needs or requirements of the Data
+Resource managed by Seedcase.
+
+The diagram below illustrates the interaction and the direction of
+information flow between the Seedcase framework and its communication
+partners.
 
 ![[C4 Context diagram](https://c4model.com/#SystemContextDiagram)
-showing users and external systems that connect to the Data Resource
-managed by Seedcase.](/design/images/context.png)
+showing users and external systems that communicate with the Seedcase
+framework.](/images/context.png)
+
+The table below indicates, in very general terms, the nature of the
+messages exchanged between the Seedcase framework and its communication
+partners. Input messages are sent by a communication partner to the
+Seedcase framework, while output messages are sent by the framework to a
+communication partner.
```

Comment:
Not sure I understand what "messages" mean here. Like "input"?

Reply:
Yeah, like input and output. I think it's supposed to state very very generally (and more conceptually than technically) what kind of communication is expected at the boundary of the system, so the type of information exchanged between a user and the system. But also happy to just drop the table if it doesn't add very much